book.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'Rental'
+
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(person, date)
+    @rentals.push(Rental.new(date, self, person))
```

Comment:
- [ ] Kindly make this function only create a new rental as you already add it to the rentals list in the Rental class initialize method. This line will make the same rental added twice to the list so that it will be like this:
```ruby

  def add_rental(person, date)
    Rental.new(date, self, person)
  end
```

Reply:
Hi, @gemmen29 Thanks fro your suggestion.
Changed the same.